packages/hardware-trezor/src/transformers/tx.ts

```diff
@@ -28,6 +31,7 @@ export const trezorTxTransformer: Transformer<
   requiredSigners: ({ requiredExtraSignatures }, context) =>
     requiredExtraSignatures ? mapRequiredSigners(requiredExtraSignatures, context!) : undefined,
   scriptDataHash: ({ scriptIntegrityHash }) => scriptIntegrityHash?.toString(),
+  tagCborSets: () => false, // Set this to true once inConwayEra is set to true in serialization classes
```

Comment:
Is it exported?
```suggestion
  tagCborSets: () => isConwayEra,
```

Reply:
The refactor shoud address this @mkazlauskas